app/src/main/java/ca/llamabagel/transpo/search/ui/SearchActivity.kt

```diff
@@ -40,19 +44,13 @@ class SearchActivity : AppCompatActivity() {
         setSupportActionBar(findViewById(R.id.toolbar))
 
         val searchBar = findViewById<CustomSearchView>(R.id.search_bar)
+        val filterBtn = findViewById<ImageButton>(R.id.filter_button)
```

Comment:
"Btn" feels like a bad abbreviation

Reply:
Lol idk it puts the equals sign in the same spot as the searchBar

Reply:
Ehh ok